CONTRIBUTING.md

```diff
@@ -19,11 +19,24 @@ only a small amount of memory.
 
 To run the Kwok container with the Kwok Kubernetes tool, follow these steps:
 
-1. Build the Kwok cluster by running the following command `cd kwok` and:
+1. login to quay by running the following command:
     ```
-    podman build --build-arg CLUSTER_NAME=<cluster-name> -t <image-name> -f Dockerfile .
+    podman login --username <your username> --password <your password> quay.io
     ```
 
+    then pull the kwok image from quay:
+    ```
+    podman pull quay.io/srivickynesh/kwok:kwok-bundle
```

Comment:
@srivickynesh we can't rely on you never changing roles or jobs - please avoid using your personal accounts or builds.

Reply:
As I have mentioned this is the temporary placeholder until we finalise on pushing this image to quay org.

Reply:
What is keeping us from putting the image is a better place to begin with? Are you lacking permissions for the `redhat-appstudio` organisation in Quay.io? 

Also to the best of my understanding this is not the vanilla kwok image but our own modified version, so we are probably better off giving it a different, more descriptive, name.

Reply:
`o11y-kwok ` sounds good? I will push it to quay.io.

Reply:
`segment-bridge-kwok` is probably better.

But it seems it may be easier to just fix the build issue

Reply:
fix: #42

Reply:
We won't need to use Quay login and pre-pull image. As per this #42.

Reply:
fixed with the relevant details